tests/stream/test_stream_smoke/0028_replay_stream.json

```diff
@@ -0,0 +1,618 @@
+                "default": [
+                    "todo",
+                    "to_support",
+                    "change",
+                    "bug",
+                    "sample",
+                    "native_not_support"
+                ],
+                "cluster": [
+                    "view",
+                    "cluster_table_bug"
+                ]
+            }
+        }
+    },
+    "comments": "Tests covering replay stream",
+    "tests": [
+        {
+            "id": 0,
+            "tags": [
+                "replay_stream"
+            ],
+            "name": "easy test replay_stream",
+            "description": "easy test replay_stream",
+            "steps": [
+                {
+                    "statements": [
+                        {
+                            "client": "python",
+                            "query_type": "table",
+                            "query": "drop stream if exists test29_stream"
+                        },
+                        {
+                            "client": "python",
+                            "query_type": "table",
+                            "exist": "test29_stream",
+                            "exist_wait": 2,
+                            "wait": 1,
+                            "query": "create stream if not exists test29_stream (id int, value int);"
+                        },
+                        {
+                            "client": "python",
+                            "query_type": "table",
+                            "wait": 1,
+                            "query": "insert into test29_stream(id, value) values (1, 2);"
+                        },
+                        {
+                            "client": "python",
+                            "query_type": "table",
+                            "wait": 2,
+                            "query": "insert into test29_stream(id, value) values (3, 2);"
+                        },
+                        {
+                            "client": "python",
+                            "query_type": "table",
+                            "wait": 1,
+                            "query": "insert into test29_stream(id, value) values (1, 2);"
+                        },
+                        {
+                            "client": "python",
+                            "query_type": "table",
+                            "wait": 2,
+                            "query": "insert into test29_stream(id, value) values (9, 2);"
+                        },
+                        {
+                            "client": "python",
+                            "query_type": "stream",
+                            "terminate": "auto",
+                            "query_end_timer": 20,
+                            "query_id": "2900",
+                            "wait": 1,
+                            "query": "select id, value from test29_stream where _tp_time > earliest_timestamp() settings replay_speed=1;"
```

Comment:
This case only check whether emit results.
we should also verify the output's interval within an reasonable range. For example: 
```sql
--- allow `< 1000ms`
select date_diff('millisecond', now64(3, 'UTC'), to_datetime64(tp_append_time/1000, 3, 'UTC')) < 1000 as verified from ...
```